pages/common/sed.md

```diff
@@ -2,21 +2,21 @@
 
 > Run replacements based on regular expressions
 
-- replace all occurrences of a string in a file, and print the result
+- replace the first occurrence of a string in a file, and print the result
 
-`sed 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed 's/{{find}}/{{replace}}/' {{filename}}`
 
-- replace all occurrences of a string in a file, and overwrite the file
+- replace all occurrences of a string in a file, overwriting the file
   contents
 
-`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed --in-place 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
> If you agree, let's go with that one.

You mean "replace all occurrences of a string in a file, overwriting the file (i.e. in-place)" rather than "Replace all occurrences of a string in place"?

If so, yeah, it isn't ideal but I don't think it's too bad. Your version is shorter but may be ambiguous. Please confirm and I'll perform the change. Btw thanks for the discussion :) (I wasn't entirely happy with my forced choice either).